legend-engine-xt-serviceStore-pure/src/main/resources/core_servicestore/executionPlan/executionPlan.pure

```diff
@@ -20,6 +20,7 @@ import meta::external::store::service::executionPlan::nodes::*;
 import meta::external::store::service::metamodel::*;
 import meta::external::store::service::metamodel::mapping::*;
 import meta::external::store::service::metamodel::runtime::*;
+import meta::pure::runtime::connection::authentication::*;
```

Comment:
lets move this import near meta::pure imports

Reply:
done